core/src/main/java/org/elasticsearch/action/bulk/BulkRequestHandler.java

```diff
@@ -71,9 +83,10 @@ public void execute(BulkRequest bulkRequest, long executionId) {
             try {
                 listener.beforeBulk(executionId, bulkRequest);
                 BulkResponse bulkResponse = Retry
-                        .on(EsRejectedExecutionException.class)
-                        .policy(backoffPolicy)
-                        .withSyncBackoff(client, bulkRequest);
+                    .on(EsRejectedExecutionException.class)
```

Comment:
Can you not do the formatting changes in future? Personally I like the 8 spaces, but regardless I think it makes the change harder to review.